docs/installation.rst

```diff
@@ -1,26 +1,16 @@
 Installation
 ============
 
-Internally in Equinor, ecl2df is distributed through Komodo.
+Internally in Equinor, ecl2df is distributed through Komodo and
+nothing is needed besides activating Komodo, typically through::
 
-You need opm-common installed upfront.
+  source /prog/res/komodo/stable/enable
 
```

Comment:
or source /prog/res/komodo/stable/enable.csh for us cshell users :-)

Reply:
added

Reply:
Or just refer to Komodo guide (so it will not give you extra work if something changes in Komodo)